c/README.md

```diff
@@ -84,14 +84,48 @@ Initialize a `blake3_hasher` in the default hashing mode.
 
 ---
 
+### Single-threaded update
+
 ```c
 void blake3_hasher_update(
   blake3_hasher *self,
   const void *input,
   size_t input_len);
 ```
 
-Add input to the hasher. This can be called any number of times.
+Add input to the hasher with single-threaded update. This can be called any number of times.
+
+### Multi-threaded update
+
+```c
+void blake3_hasher_update_tbb(
```

Comment:
@oconnor663 API design question: Do we _want_ to reference the implementation in the API name?

Reply:
Just for my own reasoning, I named it that way to be consistent with the existing Rust API:

https://docs.rs/blake3/latest/blake3/struct.Hasher.html#method.update_rayon